pkg/controllers/nodeclaim/lifecycle/liveness_test.go

```diff
@@ -141,6 +147,44 @@ var _ = Describe("Liveness", func() {
 		// If the node hasn't registered in the registration timeframe, then we deprovision the nodeClaim
 		fakeClock.Step(time.Minute * 20)
 		_ = ExpectObjectReconcileFailed(ctx, env.Client, nodeClaimController, nodeClaim)
+		nodePool = ExpectExists(ctx, env.Client, nodePool)
+		nodeRegistrationHealthySC := ExpectStatusConditionExists(nodePool, v1.ConditionTypeNodeRegistrationHealthy)
+		Expect(nodeRegistrationHealthySC.Status).To(Equal(metav1.ConditionFalse))
```

Comment:
You could consider using `ExpectStatusConditions` that we use in a few places from https://github.com/awslabs/operatorpkg/blob/main/test/expectations/expectations.go#L131